packages/hooks/src/useTrackedEffect/index.ts

```diff
@@ -0,0 +1,22 @@
+import { useEffect, useRef, DependencyList } from 'react';
+
+const useTrackedEffect = (effect, deps?: DependencyList) => {
+  const initDeps = useRef<DependencyList>();
```

Comment:
It's better to be named as `previousDepsRef`

```suggestion
  const previousDepsRef = useRef<DependencyList>();
```